notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb

```diff
@@ -4,31 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>I see the logic opens the same FITS file twice. Consider opening the file once and storing the data for both chips. </p>

<pre class="ql-syntax" spellcheck="false">with fits.open(rootname + "_flc.fits") as hdulist: chip1_data = hdulist["SCI", 2].data chip2_data = hdulist["SCI", 1].data
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>